src/components/Login/Signin.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Field from './field/field';
+import { Grid,Paper, Avatar, TextField, Button, Typography,Link } from '@material-ui/core'
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+const Signin=()=>{
+
+    const paperStyle={padding :20,height:'70vh',width:280, margin:"20px auto"}
```

Comment:
usually when using material-ui, it would look like this:

```
import {makeStyles} from '@material-ui/styles'

const useStyles = makeStyles(theme => ({
   paperStyle: {padding :20,height:'70vh',width:280, margin:"20px auto"}
   ...
})

const Signin = () => {
     const classes = useStyles()

      return (
            <Paper className={classes.paperStyle}> </Paper>
            ...
            )
}
```
^^^this is just an example of the different pieces

you can see material-ui in action in our project [here](https://github.com/Amber-Plus/Amber-Plus/blob/49ed05deb9852cbf5def950febd9c8f8df396e86/src/components/PersonAlert/PersonAlertProfile.js#L13)

Reply:
but the way you did is functional and totally works! It's just not optimal, and moving forward it's best to use makeStyles() 😄